src/app.py

```diff
@@ -783,11 +821,40 @@ def app_manifest(app):
 
     manifest, extracted_app_folder = _extract_app(app)
 
-    shutil.rmtree(extracted_app_folder)
-
     raw_questions = manifest.get("install", {}).values()
     manifest["install"] = hydrate_questions_with_choices(raw_questions)
 
+    # Add a base64 image to be displayed in web-admin
+    if Moulinette.interface.type == "api":
+        import base64
+
+        manifest["image"] = None
```

Comment:
```suggestion
        manifest["screenshots"] = None
```

Sounds odd to to name this image instead of screenshots ? :thinking: 

(In the future we may have other images such as the logo or icon so can be nice to prevent that confusion)

Reply:
renamed to `screenshot` singular